### DISCUSSION TITLE
Can I use the client server mode of UE4 to enable multiple users to control their own airsim UAV flight?

### DISCUSSION THREAD

**Discussion by Newkami** [microsoft/AirSim#4630] [Support Q&A] [answered]
Hello, now I want to make an attempt to build a multi-user UAV platform with UE4. Specifically, it is similar to the concept of multiplayer games. Each user can specify drones according to their own needs.
  Can we only use airsim's UAV model and flight function. By making some changes on the communication among drones and read settings.json . I just want to ask whether this idea is feasible, or whether someone has made some attempts.

**Comment by jonyMarino** [comment, marked as answer]
Hi @Newkami! Yeah, that is feasible You need to specify the drone name for each script.

**Reply by Newkami** [reply]
Thanks a lot.  But I still have no idea about how to specify a different setting.json for each client. I will study it in depth and look forward to discussing with you in the future。

**Reply by jonyMarino** [reply]
The settings.json is processed only by the simulator, not the clients
